open-craft/opencraft
# [BB-6228] Add Maple support

This PR adds Maple support, including changes for MFEs and required settings for [Studio Oauth](https://github.com/openedx/edx-platform/blob/open-release/maple.master/docs/guides/studio_oauth.rst).

Testing
--------
1. Deploy an OCIM server with this branch or check this branch on a running Stage environment.
2. Create a production instance either via shell or web UI.
3. Create a master instance.
4. Provision both instances.
5. Ensure deployment succeeds for both instances.
6. Make sure Studio login works.
